apps/system-e2e/src/integration/air-discount-scheme/homepage.spec.ts

```diff
@@ -0,0 +1,58 @@
+// import { idsLogin } from '../../support/commands'
+import {
+  aliasQuery,
+  getFakeUser,
+  getFamily,
+  getBaseUrl,
+  getDiscountData,
+} from '../../support/utils'
+
+describe('Home page', () => {
+  const baseUrl = getBaseUrl(Cypress.config())
+
+  const fakeUser = getFakeUser(Cypress.env('fakeUsers'), 'gervimaður afríka')
+  const family = getFamily(fakeUser)
+
+  beforeEach(() => {
+    cy.idsLogin({
+      phoneNumber: fakeUser.phoneNumber,
+      urlPath: '/min-rettindi',
+    })
+    cy.intercept('POST', `${baseUrl}/api/graphql`, (req) => {
+      aliasQuery(req, 'DiscountsQuery')
+    })
```

Comment:
```suggestion
    cy.intercept('POST', `${baseUrl}/api/graphql`).as('DiscountsQuery')
```

Reply:
this is intentional.

See the following

```javascript
export const hasOperationName = (
  req: CyHttpMessages.IncomingHttpRequest,
  operationName: string,
) => {
  const { body } = req
  return typeof body === 'object' && body.operationName === operationName
}

export const aliasQuery = (
  req: CyHttpMessages.IncomingHttpRequest,
  operationName: string,
) => {
  if (hasOperationName(req, operationName)) {
    req.alias = operationName
  }
}
```